.kokoro/docker/cloudbuild-test.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 7200s # 2 hours
+substitutions:
+  _JAVA_SHARED_CONFIG_VERSION: '1.5.7'
+  _GRAALVM_22_3_JDK11_ALIAS: 'graalvm_a'
+  _GRAALVM_22_3_JDK17_ALIAS: 'graalvm_b'
+steps:
+  # GraalVM build
+  - name: gcr.io/cloud-builders/docker
+    args: ["build", "-t", "gcr.io/$PROJECT_ID/${_GRAALVM_22_3_JDK11_ALIAS}:${_JAVA_SHARED_CONFIG_VERSION}", "."]
```

Comment:
Is the aliasing in the right direction to make maintenance as simple as possible? Updating the version associated with graalvm_a is currently going to require edits throughout the file. Can we refactor so that 22.3 and JDK11 are defined in the minimal places such that updating the mapping is as few edits as possible?

This top set of steps will likely always be dedicated to graalvm_a regardless of which version of GraalVM+JDK we apply to that label, so if we flipped the indirection we could update graalvm_a's mapping in a single line update.
```
_GRAALVM_A: 'graalvm22.3_jdk11' // We can now update this single line and never touch the graalvm_a steps.
...
gcr.io/$PROJECT_ID/graalvm_a:${_JAVA_SHARED_CONFIG_VERSION}
...
dir: .kokoro/docker/${_GRAALVM_A}
...
  - name: gcr.io/gcp-runtimes/structure_test
    args:
      ["-i", "gcr.io/$PROJECT_ID/graalvm_a:${_JAVA_SHARED_CONFIG_VERSION}", "--config", ".kokoro/docker/${_GRAALVM_A}.yaml", "-v"]
    waitFor: ["graalvm-build"]
  - name: gcr.io/cloud-builders/docker
    args:
      [
          "tag",
          "gcr.io/$PROJECT_ID/graalvm_a:${_JAVA_SHARED_CONFIG_VERSION}",
          "gcr.io/cloud-devrel-public-resources/graalvm_a:${_JAVA_SHARED_CONFIG_VERSION}",
      ]
```